src/Settings/Settings.scss

```diff
@@ -0,0 +1,9 @@
+.Settings {
+  color: #1b2431;
```

Comment:
for information we begin with `flex` and after the `width and height`, `font` and `color`